types/ttf2woff/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for ttf2woff 2.0
+// Project: https://github.com/fontello/ttf2woff#readme
+// Definitions by: Kaspar Vollenweider <https://github.com/casaper>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+declare function ttf2woff(ttf: Uint8Array): Buffer;
```

Comment:
Shouldn't optional options appear here?
https://github.com/fontello/ttf2woff/blob/master/index.js#L108
the very existence of optiona `metadata` impacts outcome IMO:
https://github.com/fontello/ttf2woff/blob/master/index.js#L255

Reply:
@peterblazejewicz 
You're right. I'll have to fix this. 

It was premature to make a PR with this. Thanks for spotting!

Reply:
@peterblazejewicz 

Fixed that now. The package itself lacks the documentation regarding this completely.

I added the documentation from W3 as doc comment now, on a minimal basis.

Hope that is enough like that.

Thx.